newIDE/README-themes.md

```diff
@@ -0,0 +1,38 @@
+# GDevelop Themes
+
+Themes are a consistent set of colors and other styles that unify the look and feel of the application.  
+
+## How themes are handled
+
+Theme styles are held in a `theme.json`, which is used to generate:
+ - A `.css` file, containing the styles as CSS Custom Properties. These properties are set by applying the theme class 
+ to specific sections of the app. To see where the properties are used, see the stylesheets in the `Global` folder.
+- A `.json` file, containing the styles as a flat object, where each style is a field. This is imported and used by the Material UI library
+as well as custom components
+
+They are created by leveraging (Style Dictionary)[https://amzn.github.io/style-dictionary] to compile them from `theme.json` using 
+the `compile-theme-style` script, in the scripts folder. There is also an npm script to run it (`build-theme-resources`)
```

Comment:
As mentioned in the package.json, let's use the same name for the script and the npm script?

Reply:
done ✅